internal/upload/album_test.go

```diff
@@ -98,3 +118,121 @@ func TestAlbumNameUsingFolderName(t *testing.T) {
 		assert.Equal(t, tt.out, albumNameUsingFolderName(tt.in))
 	}
 }
+
+func TestGetTemplateFunctionName(t *testing.T) {
+	var testData = []struct {
+		in  string
+		out string
+	}{
+		{in: "", out: ""},
+		{in: "$cutLeft(", out: "cutLeft"},
+		{in: "$cutLeft (", out: ""},
+		{in: "cutLeft(", out: ""},
+		{in: "$cutLeft(anything", out: "cutLeft"},
+	}
+
+	for _, tt := range testData {
+		assert.Equal(t, tt.out, getTemplateFunctionName(tt.in))
+	}
+}
+
+func TestParseAlbumNameTampleWithTokens(t *testing.T) {
```

Comment:
Maybe a typo: `Tample` instead of `Template`?

Reply:
yeah will correct